smartBugs.py

```diff
@@ -15,47 +17,51 @@
 from src.output_parser.SarifHolder import SarifHolder
 from time import time, localtime, strftime
 
+SMARTBUGS_FILE_PATH = os.path.dirname(os.path.realpath(__file__))
+DATASET_CONFIG_FILE_PATH = SMARTBUGS_FILE_PATH + '/config/dataset/dataset.yaml'
 
-cfg_dataset_path = os.path.abspath('config/dataset/dataset.yaml')
+cfg_dataset_path = os.path.abspath(DATASET_CONFIG_FILE_PATH)
 with open(cfg_dataset_path, 'r') as ymlfile:
     try:
         cfg_dataset = yaml.safe_load(ymlfile)
     except yaml.YAMLError as exc:
-        print(exc)
+        raise Exception('Dataset YAML not found.'
+                        ' Please check that the following file is correct at smartbugs/config/dataset/dataset.yaml')
 
 output_folder = strftime("%Y%d%m_%H%M", localtime())
```

Comment:
nitpicking: rename variable to `log_output_folder`

Reply:
It's also used for the results folder, but I agree that it should be better named. Maybe runtime_output_folder? And the analyse_files function receives this variable as "now". Should I changed them both?